common/src/Microsoft.Azure.IIoT.Abstractions/src/PcsVariable.cs

```diff
@@ -294,6 +294,6 @@ public static class PcsVariable {
             "PCS_LOGS_PATH";
         /// <summary> The maximum size of the (IoT D2C) message egress queue </summary>
         public const string PCS_MAX_EGRESS_MESSAGE_QUEUE =
```

Comment:
Please rename PcsVariable as well. Having PcsVariable with a value different from its name is very confusing.

Reply:
The idea is here to just rename the external API surface to avoid breaking changes. Everything else will remain as egress as there is outgress.

Reply:
I understand that, I see that internal models still have `MaxEgressMessageQueue`. What I am asking for is that the PCS variable has the same name as its value. PcsVariable is for having a centralized location for storing environment variables that we use for configuration. So please only rename `PcsVariable.PCS_MAX_EGRESS_MESSAGE_QUEUE` to `PcsVariable.PCS_DEFAULT_PUBLISH_MAX_OUTGRESS_MESSAGES` to reflect the value that we will expect from environment, which is `PCS_DEFAULT_PUBLISH_MAX_OUTGRESS_MESSAGES`.